.prettierignore

```diff
@@ -1,5 +1,4 @@
 *.less
 node_modules
-src/components/icons/*.svelte
```

Comment:
Should we add this back in now that the icons are regenerated? It might be a lot of prettier has to check these files every time, however it’s probably fine though.